### PR TITLE
Add folder for ESW

### PR DESCRIPTION
**Pull request by prapalu**
Add folder for ESW

### CODE REVIEW

**Comment by dgarijo** [comment]
@prapalu please add yourself a a maintainer in the readme file

File: esw/.htaccess [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+
+# Rewrite engine setup
+RewriteEngine On
+RewriteBase /def
```
Comment (davidlehn) [inline esw/.htaccess:16]: It doesn't look like this is used?

File: esw/readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# ESW Ontology: Exploratory Search Workflow
+
+This [W3ID](https://w3id.org) provides a persistent URI namespace for Exploratory Search Workflow (ESW) resources.
+
+## Contact
+This space is administered by:  
+
+**Gianmarco Prando**  
+*Computer Engineer *
+Padua, Italy
```
Comment (TallTed) [inline esw/readme.md:10]: To make the markup and line breaks behave as you intended... you'll want to make another PR, with these lines --
```suggestion
**Gianmarco Prando**  
*Computer Engineer*  
Padua, Italy  
```
